packages/localizations/src/en-US.ts

```diff
@@ -487,6 +544,11 @@ export const enUS: LocalizationResource = {
     form_username_invalid_length: '',
     identification_deletion_failed: 'You cannot delete your last identification.',
     not_allowed_access: '',
+    organization_domain_blocked: '',
+    organization_domain_common: 'Esse é um domínio comum, por favor use um diferente.',
+    organization_membership_quota_exceeded: '',
```

Comment:
Is this expected to be empty ?

Reply:
I think we should leave it empty in the English translation to use the error original message from FAPI, I think we use this logic for some other keys.

Reply:
The downside tho is that it's difficult for external contributors to figure out what key should be updated for their language.

Reply:
Updated https://github.com/clerk/javascript/pull/4123/commits/7ef029d3c99fcfc36d30a0ec7804f0c623d6ac40